.github/components.yml

```diff
@@ -141,6 +143,13 @@ PyTorch_FE:
     - Python_API
     - TOKENIZERS
 
+JAX_FE:
+  revalidate:
+    - MO
```

Comment:
do we need it?
I suppose it's not enabled via MO ?

Reply:
MO mark also corresponds to ovc tool. It is enabled via ovc and ovc.convert_model.